chain/ethereum.ts

```diff
@@ -360,6 +360,44 @@ export namespace ethereum {
     ) {}
   }
 
+  /**
+   * An Ethereum transaction receipt.
+   */
+  export class TransactionReceipt {
+    constructor(
+      public transaction_hash: Bytes,
+      public transaction_index: BigInt,
+      public block_hash: Bytes,
+      public block_index: BigInt,
+      public cumulative_gas_used: BigInt,
+      public gas_used: BigInt,
+      public contract_address: Address,
+      public logs: Array<Log>,
+      public status: BigInt,
+      public root: Bytes,
+      public logs_bloom: Bytes,
+    ) {}
+  }
+
+  /**
+   * An Ethereum event log.
+   */
+  export class Log {
+    constructor(
+      public address: Address,
+      public topics: Array<Bytes>,
+      public data: Bytes,
+      public block_hash: Bytes,
+      public block_number: Bytes,
+      public transaction_hash: Bytes,
+      public transaction_index: BigInt,
+      public log_index: BigInt,
+      public transaction_log_index: BigInt,
+      public log_type: string,
+      public removed: boolean,
```

Comment:
I think those should use `camelCase` instead of `snake_case`

Reply:
Thanks! Fixed in https://github.com/graphprotocol/graph-ts/commit/b43da49